cmds/bmp/status_command.go

```diff
@@ -56,11 +57,8 @@ func (cmd statusCommand) Execute(args []string) (int, error) {
 		return 1, err
 	}
 
-	cmd.ui.Println("BMP server info")
-	cmd.ui.Printf(" status: %d", info.Status)
-	cmd.ui.Printf(" name: %s", info.Data.Name)
-	cmd.ui.Printf(" version: %s", info.Data.Version)
-	cmd.ui.Println()
+	fmt.Println(" name:    ", info.Data.Name)
```

Comment:
Let's discuss this change. The cmd.ui is meant to encapsulate all output and I feel if you are doing this due to debugging then we may need to fix the `Ui` interface